app/config/messaging.js

```diff
@@ -0,0 +1,53 @@
+const joi = require('joi')
+const { PRODUCTION } = require('../constants/environments')
+
+const schema = joi.object({
+  messageQueue: {
+    host: joi.string().default('localhost'),
+    useCredentialChain: joi.bool().default(false),
+    type: joi.string(),
+    appInsights: joi.object(),
+    username: joi.string(),
+    password: joi.string()
+  },
+  sendTopic: {
+    address: joi.string()
+  },
+  eventsTopic: {
+    address: joi.string()
+  }
+})
+
+const config = {
+  messageQueue: {
+    host: process.env.MESSAGE_QUEUE_HOST,
+    useCredentialChain: process.env.NODE_ENV === PRODUCTION,
+    type: 'Topic',
+    appInsights: process.env.NODE_ENV === PRODUCTION ? require('applicationinsights') : undefined,
+    username: process.env.MESSAGE_QUEUE_USER,
+    password: process.env.MESSAGE_QUEUE_PASSWORD
+  },
+  sendTopic: {
+    address: process.env.FILESEND_TOPIC_ADDRESS
+  },
```

Comment:
eventTopic no longer used in Azure as replaced with eventsTopic.  Can delete this.